### DISCUSSION TITLE
[@types/sharp] lacks latest error handling options

### DISCUSSION THREAD

**Discussion by wmelton** [DefinitelyTyped/DefinitelyTyped#61141] [Issues with a @types package]
The latest version of sharpjs has changed the constructor options by deprecating the `failOnError` option in favor of the new `failOn` syntax. 

This change is not reflected in the @latest type definitions: 
<img width="897" alt="image" src="https://user-images.githubusercontent.com/1544125/177873507-ff9673c2-e6d6-405d-9be8-453b1f830a09.png">

<img width="1088" alt="image" src="https://user-images.githubusercontent.com/1544125/177873519-10de6429-e785-4c04-af56-94b3ff883f0f.png">

[a comment by an automated account is not shown]

**Comment by ThaDaVos** [comment]
Above is still the case, when can we expect this to be fixed?

**Reply by fdebijl** [reply]
#61555

**Comment by fdebijl** [comment]
Should be fixed in #61555, along with some other missing types

![image](https://user-images.githubusercontent.com/18357629/182618758-211e304b-9632-4346-bc92-f9fa7761545d.png)
